src/main/resources/perforce/ReleaseTrigger.py

```diff
@@ -6,32 +6,43 @@
 
 import sys
 from com.perforce.p4java.server import ServerFactory
+from com.perforce.p4java.option.server import GetChangelistsOptions
+from com.perforce.p4java.core import IChangelist
+from com.perforce.p4java.core.file import FileSpecBuilder
+from com.perforce.p4java import PropertyDefs
+from java.lang import System
 
 if perforceServer is None:
-   print "No server provided"
    sys.exit(1)
```

Comment:
Great rewrite!  Thanks for the effort and the new features.  I like printing a message here since it's easy to forget to pick an item from the server dropdown (at least for me).